src/settings/_spacing.scss

```diff
@@ -1,88 +1,80 @@
-// Single points on spacing scale
-$govuk-spacing-scale-1: 5px;
-$govuk-spacing-scale-2: 10px;
-$govuk-spacing-scale-3: 15px;
-$govuk-spacing-scale-4: 20px;
-$govuk-spacing-scale-5: 25px;
-$govuk-spacing-scale-6: 30px;
-$govuk-spacing-scale-7: 40px;
-$govuk-spacing-scale-8: 50px;
-$govuk-spacing-scale-9: 60px;
+////
+/// @group settings/spacing
+////
 
-// Responsive spacing maps
-//
-// These definitions are used to generate responsive spacing that adapts
-// according to the breakpoints (see 'helpers/spacing'). These maps should be
-// used wherever possible to standardise responsive spacing.
+/// Single point spacing variables. Access using `govuk-spacing()`
+/// (see `helpers/spacing`).
+///
+/// @type Map
+/// @access private
 
-// You can define different behaviour on tablet and desktop. The 'null'
-// breakpoint is for mobile.
+$govuk-spacing-points: (
+  0: 0,
+  1: 5px,
+  2: 10px,
+  3: 15px,
+  4: 20px,
+  5: 25px,
+  6: 30px,
+  7: 40px,
+  8: 50px,
+  9: 60px
+) !default;
 
-// To use the responsive spacing maps, combine them with
-// 'govuk-responsive-margin' or 'govuk-responsive-padding' mixins
-// (see 'helpers/spacing').
+/// Responsive spacing maps
+///
+/// These definitions are used to generate responsive spacing that adapts
+/// according to the breakpoints (see 'helpers/spacing'). These maps should be
+/// used wherever possible to standardise responsive spacing.
 
```

Comment:
Picky, but can we add the `///` to the blank lines to so that it reads as one connected comment block?